src/tests/unit/tests/popup/components/__snapshots__/popup-view.test.tsx.snap

```diff
@@ -70,44 +70,65 @@ exports[`PopupView renderFailureMsgPanelForChromeUrl 1`] = `
 `;
 
 exports[`PopupView renderFailureMsgPanelForFileUrl 1`] = `
-<div
-  className="ms-Fabric unsupported-url-info-panel"
->
-  <Header
-    title="test title"
-  />
-  <div
-    className="ms-Grid main-section"
-  >
-    <div
-      className="launch-panel-general-container"
-    >
-      Your browser settings don't allow Accessibility Insights for Web to run on file URLs.
-    </div>
-    <div>
-      <div>
-        To allow this extension to run on file URLs:
-      </div>
-      <div>
-        1. Open 
-        <NewTabLink
-          aria-label="open test title extension page"
-          onClick={[Function]}
-        >
-          test title extension page
-        </NewTabLink>
-        .
-      </div>
-      <div>
-        2. Enable 
-        <span
-          className="ms-fontWeight-semibold"
-        >
-          Allow Access to file URLs
-        </span>
-        .
-      </div>
-    </div>
-  </div>
-</div>
+<FileUrlUnsupportedMessagePanel
+  deps={
```

Comment:
Would it make sense to suppress the deps value from the snapshot, since the intention of deps is that the thing under test here shouldn't necessarily care about the specific dependencies its children need? If you wanted to do it, I think it would look amount to changing L269 of `popup-view.test.tsx` to the following:

```ts
        expect(wrapped.getElement()).toMatchSnapshot({ props: { deps: expect.any(Object) } });
```

Reply:
TL;DR: changing out we handle snapshots is out of scope for this PR.

As we talked yesterday, it seems we need to run some small investigation around jest snapshot serializers. That looks promising as it will allow us to simplify snapshots (by declutering deps serialization) without having to change every test in order to do so.
That being said, the current practice on our snapshot testing is to include everything we get from calling `.getElemennt()` on the enzyme shallow wrapper, including deps so I'll be hesitant to include this change as part of this PR as it seems out of scope (the scope being extract a component that, from the `PopupView` point of view, is an implementation detail and don't really belong in the original component).